components/automate-ui/src/app/modules/infra-proxy/cookbook-details/cookbook-details.component.ts

```diff
@@ -1,18 +1,24 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
-import { Store } from '@ngrx/store';
 import { HttpClient } from '@angular/common/http';
+import { Store } from '@ngrx/store';
 import { environment as env } from 'environments/environment';
 import { Subject, combineLatest } from 'rxjs';
 import { first, filter, takeUntil, pluck } from 'rxjs/operators';
+import { identity, isNil } from 'lodash/fp';
+import { CollapsibleListMapper } from 'app/helpers/infra-proxy/collapsible-list-mapper';
```

Comment:
Please add a blank line above this one, separating 3rd-party resources from internal resources.
Also move the environment (L4) into the internal resources group.